src/main/java/com/northconcepts/datapipeline/examples/cookbook/ConfigureInsertStrategyInJdbcMultiWriter.java

```diff
@@ -0,0 +1,61 @@
+package com.northconcepts.datapipeline.examples.cookbook;
+
+import com.mysql.cj.jdbc.MysqlDataSource;
+import com.northconcepts.datapipeline.core.DataReader;
+import com.northconcepts.datapipeline.core.DataWriter;
+import com.northconcepts.datapipeline.excel.ExcelDocument;
+import com.northconcepts.datapipeline.excel.ExcelReader;
+import com.northconcepts.datapipeline.internal.jdbc.JdbcFacade;
+import com.northconcepts.datapipeline.jdbc.JdbcConnectionFactory;
+import com.northconcepts.datapipeline.jdbc.JdbcMultiWriter;
+import com.northconcepts.datapipeline.jdbc.insert.MySqlPreparedStatementInsert;
+import com.northconcepts.datapipeline.job.Job;
+
+import javax.sql.DataSource;
+import java.io.File;
+
+public class ConfigureInsertStrategyInJdbcMultiWriter {
+
+    private static final String SERVER_NAME = "localhost";
+    private static final String DATABASE_NAME = "datapipeline";
+    private static final String DATABASE_USERNAME = "etl";
+    private static final String DATABASE_PASSWORD = "etl";
+    private static final String DATABASE_TABLE = "inbound_calls";
+
+    public static void main(String[] args) {
+        MysqlDataSource dataSource = new MysqlDataSource();
+
+        dataSource.setServerName(SERVER_NAME);
+        dataSource.setDatabaseName(DATABASE_NAME);
+        dataSource.setUser(DATABASE_USERNAME);
+        dataSource.setPassword(DATABASE_PASSWORD);
+
+        createTable(dataSource);
+
+        ExcelDocument doc = new ExcelDocument();
+        DataReader reader = new ExcelReader(doc.open(new File("example/data/input/call-center-inbound-call-2.xlsx")))
```

Comment:
Let's move the `.open(new File("example/data/input/call-center-inbound-call-2.xlsx")` call to the line above.  We don't want anyone copying this code to read multiple sheets in the same Excel to open the doc each time.